lerobot-web-fe/src/components/ChatHistory.module.css

```
@@ -0,0 +1,38 @@
+.chatHistory {
+  pointer-events: all;
+  user-select: text;
+  padding: 12px 8px 10px 8px;
+  margin-bottom: -8px;
+  backdrop-filter: blur(10px);
+  border-radius: 10px 10px 0 0;
+  background-color: #00000030;
+  border-bottom-width: 0;
+  background-color: #f0f0f091;
+  border: 1px solid #fff;
```

Biome (2.1.2)

[error] Unexpected shorthand property border after border-bottom-width



(lint/suspicious/noShorthandPropertyOverrides)